pkg/operation/common/types.go

```diff
@@ -290,6 +290,12 @@ const (
 	// SecretRefChecksumAnnotation is the annotation key for checksum of referred secret in resource spec.
 	SecretRefChecksumAnnotation = "checksum/secret.data"
 
+	// ShootAlphaScalingAPIServerSize is a constant for an annotation on the shoot stating the initial API server size.
+	// Possible values are [small, medium, large, xlarge, 2xlarge].
+	// Note that this annotation is alpha and can be removed anytime without further notice. Only use it if you know
+	// what you do.
+	ShootAlphaScalingAPIServerSize = "alpha.kube-apiserver.scaling.shoot.gardener.cloud/size"
```

Comment:
why not
```suggestion
	ShootAlphaScalingAPIServerSize = "alpha.kube-apiserver.scaling.shoot.gardener.cloud/class"
```

Reply:
Makes sense, thanks. I was first using size but later decided to call it 'class' and forgot to rename it. PTAL again :)